modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -127,6 +127,10 @@ const ControlSelection = (selection: EditorSelection, editor: Editor): ControlSe
       return false;
     }
 
+    if (dom.hasClass(elm, 'mce-placeholder')) {
```

Comment:
I guess we decided to remove this but if we want to have it optional or experimental I wonder if we should use the method you had before where it specifically checked if the `data-mce-resize` was set to true it could do that and then also check if that element is the `tiny-uploadcare-placeholder` element maybe?
It becomes hard coded plugin specific things in the core but we already do that for other things so maybe that ship has already sailed.

Reply:
I will remove it for now, but we can revisit it if we decide to go with an optional resize. To be honest, I don't like adding more plugin specific stuff to core